ai-platform-unified/notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<p>"You'll do this using the pipeline defined in the previous section. " -&gt;</p><p>"You do this using the pipeline you already defined".</p><p> </p><p>"Note also that you may need" -&gt;</p><p>Note: You need</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>